db/migrate/20200615134519_add_title_and_description_to_saved_scenarios.rb

```diff
@@ -0,0 +1,45 @@
+class AddTitleAndDescriptionToSavedScenarios < ActiveRecord::Migration[5.2]
+  def up
+    add_column :saved_scenarios, :title, :string
+    add_column :saved_scenarios, :description, :text
+
+    # Get current titles and descriptions from ETEngine
+    saved_scenarios = SavedScenario.all
+    total = saved_scenarios.size
+    done = 0
+    failed = 0
+    puts "Updating #{total} saved scenarios by extracting details from ETEngine"
+
+    # ETEngine only allows batches of maximum length 2048
+    saved_scenarios.in_groups_of(2048, false) do |chunked_saved_scenarios|
+      SavedScenario.batch_load(
+        chunked_saved_scenarios,
+        params: { detailed: true }
```

Comment:
While investigating the 2048 limit, I found that `params` should be `query`:

```suggestion
        query: { detailed: true }
```

Reply:
Yes, I found it as well while doing my little benchmark. I just didn't push the commit yet 😬 sorry if it caused you any trouble